packages/web/src/components/collection/CollectionCard.test.tsx

```diff
@@ -20,42 +21,45 @@ function renderCollectionCard() {
         element={<Text variant='heading'>Test User Page</Text>}
       />
     </Routes>,
-    {
-      reduxState: {
-        collections: {
-          entries: {
-            1: {
-              metadata: {
-                playlist_id: 1,
-                playlist_name: 'Test Collection',
-                playlist_owner_id: 2,
-                permalink: '/test-user/test-collection',
-                repost_count: 10,
-                save_count: 5,
-                _cover_art_sizes: {
-                  [SquareSizes.SIZE_150_BY_150]: 'image-small.jpg',
-                  [SquareSizes.SIZE_480_BY_480]: 'image-medium.jpg'
+    merge(
+      {
+        reduxState: {
+          collections: {
+            entries: {
+              1: {
+                metadata: {
+                  playlist_id: 1,
+                  playlist_name: 'Test Collection',
+                  playlist_owner_id: 2,
+                  permalink: '/test-user/test-collection',
+                  repost_count: 10,
+                  save_count: 5,
+                  _cover_art_sizes: {
+                    [SquareSizes.SIZE_150_BY_150]: 'image-small.jpg',
+                    [SquareSizes.SIZE_480_BY_480]: 'image-medium.jpg'
+                  }
                 }
               }
             }
-          }
-        },
-        users: {
-          entries: {
-            2: { metadata: { handle: 'test-user', name: 'Test User' } }
+          },
+          users: {
+            entries: {
+              2: { metadata: { handle: 'test-user', name: 'Test User' } }
+            }
           }
         }
-      }
-    }
+      },
+      options
+    )
   )
 }
 
 describe('CollectionCard', () => {
-  it('renders a button with the label comprising the collection and artist name', () => {
+  it('renders a button with the label comprising the collection and artist name, and favorites and reposts', () => {
     renderCollectionCard()
     expect(
       screen.getByRole('button', {
-        name: /test collection test user/i
+        name: /test collection test user reposts 10 favorites 5/i
```

Comment:
Man I wish there was a way to interpolate the `aria-labelledby` to inject stuff like `test collection **by** test user **with**` but I dont think this is possible without just adding those directly into the labels I guess

Reply:
yeah agreed, fwiw this pattern is what spotify uses, it doesn't have the "by" etc